rest/admin_api_test.go

```diff
@@ -3705,3 +3705,42 @@ func TestConfigsIncludeDefaults(t *testing.T) {
 	runtimeServerConfigDatabase = runtimeServerConfigResponse.Databases["db2"]
 	assert.True(t, runtimeServerConfigDatabase.Unsupported.DisableCleanSkippedQuery)
 }
+
+func TestLegacyCredentialInheritance(t *testing.T) {
+	if base.UnitTestUrlIsWalrus() {
+		t.Skip("This test only works against Couchbase Server")
+	}
+	defer base.SetUpTestLogging(base.LevelInfo, base.KeyHTTP)()
```

Comment:
Can probably move this now you have test working? Presume this was for debugging

Reply:
Seems like this is needed so logging can be teardown due to setupServerContext setting up logging